src/ACEtk/interpretation/ContinuousEnergyNeutron/Builder/EnergyDistribution.hpp

```diff
@@ -0,0 +1,46 @@
+struct EnergyDistribution{
```

Comment:
This is not under Reaction because it can appear in multiple spots (photon production)?

Reply:
as well as delayed neutron precursors